### PR TITLE
Review engineering challenge

### PR DESCRIPTION
**Pull request by Partins**

### CODE REVIEW

File: report/finished/chapters/engineering_challenge.tex [hunk 1]
```diff
@@ -0,0 +1,8 @@
+To evaluate the performance of the robot in terms of completing the engineering challenge a setupr shown in Fig. \ref{fig:factory_setup}. The setup consists of the main factory model and red lines connecting the drop-off point to the left and the pickup point to the right. The read lines specify the path the robot can take. At each endpoint and corner of the colored path a QR-code is placed which contains an coordinate of that point in space. The goal for the robot is to receive an instruction from Arrowhead, either to pickup the piece on the platform or drop it off in the drop off area. The robot would then navigate autonomously to the correct coordinate and perform the pick up or drop off action with the arm.
```
Comment (NiklasDahlquist1) [inline report/finished/chapters/engineering_challenge.tex]: setupr -> setup*
"the read lines" -> "the **red** lines"
"drop it off **at** the drop off area", Not sure what sounds better?

File: report/unsafe/sections/martin_engineering_challenge.tex [hunk 2]
```diff
@@ -0,0 +1,8 @@
+To evaluate the performance of the robot in terms of completing the engineering challenge a setupr shown in Fig. \ref{fig:factory_setup}. The setup consists of the main factory model and red lines connecting the drop-off point to the left and the pickup point to the right. The read lines specify the path the robot can take. At each endpoint and corner of the colored path a QR-code is placed which contains an coordinate of that point in space. The goal for the robot is to receive an instruction from Arrowhead, either to pickup the piece on the platform or drop it off in the drop off area. The robot would then navigate autonomously to the correct coordinate and perform the pick up or drop off action with the arm.
```
Comment (NiklasDahlquist1) [inline report/unsafe/sections/martin_engineering_challenge.tex:1]: Same comments as the file in /finnished/chapters.

File: report/unsafe/sections/martin_evaluation_challenge.tex [hunk 3]
```diff
@@ -0,0 +1,14 @@
+\subsection{Engineering challenge}
+Performance of the robot during navigation on the setup shown in Fig. \ref{fig:factory_setup} was evaluated with the following criterias
```
Comment (NiklasDahlquist1) [inline report/unsafe/sections/martin_evaluation_challenge.tex:2]: End sentence, I think?

File: report/unsafe/sections/martin_evaluation_challenge.tex [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+\begin{itemize}
+    \item Autonomy of the robot
+    \item Speed and navigation
+    \item Pickup / drop-off performance
+\end{itemize}
+
+The test were performedd at Luleå University of Technology at EISLAB with a local Arrowhead cloud and private network. The robot was able to navigate the factory floor and perform the action but moved slowly. This is because the Dynamixel motors used to translate the robot are slow. Communicating with Arrowhead was working and the test was proved successfull. A video with the whole run can be seen here: https://www.youtube.com/watch?v=YoAFspEC2no.
```
Comment (NiklasDahlquist1) [inline report/unsafe/sections/martin_evaluation_challenge.tex:10]: "are slow" -> "can only achieve a low rpm"
slow sounds so negative :)
(and might be to general)

Add hyperlink and (maybe) use shortened URL: https://youtu.be/YoAFspEC2no

File: report/unsafe/sections/martin_evaluation_challenge.tex [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+We can't do the evaluation without mentioning som limitations. The speed of the robot is mentioned above but also because of the line following algorithm the robot can get lost if it ever loses the line. This could be prevented with an external navigation system such as GPS, Ultra Wide Band (UWB) or simliar. 
+
+\subsection{Factory setup}
+In this project we also propose how a factory test-bed can be designed. In this solution the main components are the colored lines and QR-codes. The ain advantage of this setup is that it's very flexible. First of all the path can have any shape and the QR-codes can contain any information. Further, more colors can be utilized to identify different parts of the factory, stop certain robots from following them or use lines with multiple colors to determine the orientation of the robot while following the path. While flexible thhis solution may get complicated and hard to alter for very big factories. Also a fast and easy system to register QR-codes and information would need to be implemented.
```
Comment (NiklasDahlquist1) [inline report/unsafe/sections/martin_evaluation_challenge.tex:14]: ain -> main
I am not sure **it's** should be replaced with **it is**?
thhis -> this

**Comment by Partins** [comment]
seems like a little too many files are in this commit but I changed in the chapter file

**Pull request merged**